en/messages/README.md

```diff
@@ -1,5 +1,5 @@
 # Message Definitions
 
-MAVLink messages are defined in an [XML file](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml). This XML file is located in the [message definition](https://github.com/mavlink/mavlink/blob/master/message_definitions/) repository. Only messages contained in this file are considered standard messages.
+MAVLink messages are defined in XML files in the [mavlink/message definitions](https://github.com/mavlink/mavlink/blob/master/message_definitions/) folder. The messages that are common to all systems are defined in [common.xml](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml) (only messages contained in this file are considered standard messages). MAVLink protocol-specific messages and vendor-specific messages are stored in separate XML files.
```

Comment:
@LorenzMeier FYI, I tried to make the explanation of what messages are for more clear.